src/Catalyst.Core.Modules.Dfs.Tests/IntegrationTests/CoreApi/ObjectApiTest.cs

```diff
@@ -0,0 +1,224 @@
+#region LICENSE
+
+/**
+* Copyright (c) 2019 Catalyst Network
+*
+* This file is part of Catalyst.Node <https://github.com/catalyst-network/Catalyst.Node>
+*
+* Catalyst.Node is free software: you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation, either version 2 of the License, or
+* (at your option) any later version.
+*
+* Catalyst.Node is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+* GNU General Public License for more details.
+*
+* You should have received a copy of the GNU General Public License
+* along with Catalyst.Node. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#endregion
+
+using System;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Catalyst.Abstractions.Dfs;
+using Catalyst.Abstractions.Hashing;
+using Catalyst.Core.Lib.Dag;
+using Catalyst.Core.Modules.Dfs.Tests.Utils;
+using Catalyst.Core.Modules.Hashing;
+using MultiFormats.Registry;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Catalyst.Core.Modules.Dfs.Tests.IntegrationTests.CoreApi
+{
+    public class ObjectApiTest
+    {
+        private readonly IHashProvider _hashProvider;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the field '_hashProvider' and declare it as a local variable in the relevant methods.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)